### PR TITLE
Translated start section, intro to the medical report

### PR DESCRIPTION
**Pull request by jeffmaher**
# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates the start section to the application.

# Test Instructions

1. Go to the application
1. If your browser is in EN-xx locale, click `Français` at the top
1. See French translation (original EN and FR forms are linked in the Trello card)

# Other Notes

There's text that says write stuff at the top of every page, which is obviously no longer applicable and is a different scope than this task. We can change that in a later pull request / [Trello task](https://trello.com/c/WqJVeqsh/356-misc-content-tasks). That might be a good first real change to the medical report content as practice PR reviews with ESDC folks.

### CODE REVIEW

**Comment by jeffmaher** [comment]
@keithiopia : Could you review, then pass to Calvin or Dave?

**Comment by jeffmaher** [comment]
@keithiopia mentioned that the "Let's Go" button was not translated. Just updated with a fix for that. Ready for re-review.

File: locales/fr.json [hunk 1]
```diff
@@ -45,8 +45,7 @@
   "phase.desc": "This is a test site and should NOT be used to apply for this service.",
```
Comment (keithiopia) [inline locales/fr.json]: This needs updating to something like "Ce site changera au fur et à mesure que nous testerons les idées."

Comment (jeffmaher) [reply locales/fr.json]: Good find! Updated with that message.

**Review by keithiopia — APPROVED** [review]
Looks a good match to me.
Your turn, @dsamojlenko .

**Review by dsamojlenko — APPROVED** [review]
LGTM 🌮

**Pull request merged**